app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -304,6 +328,7 @@ public Survey closeSurvey(GuidCreatedOnVersionHolder keys) {
         Survey existing = getSurvey(keys);
         existing.setPublished(false);
         existing.setModifiedOn(DateUtils.getCurrentMillisFromEpoch());
+        existing.setSchemaRevision(null);
```

Comment:
Alx updated the survey API so there's no longer a "close", so this is moot. Once published, surveys can only be versioned (which creates a new survey version entirely) or deleted (which marks the survey version as deleted, but still exists in the database so users can submit answers to it).